src/resources/Conversion/GeneralPartnershipResource.ts

```diff
@@ -0,0 +1,39 @@
+import { CorrectionTypes, FilingCodes, NameRequestEntityTypes } from '@/enums/'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module/'
+import { ResourceIF } from '@/interfaces/'
+
+export const GeneralPartnershipResource: ResourceIF = {
+  entityType: CorpTypeCd.PARTNERSHIP,
+  entityReference: 'Business',
+  contactLabel: 'Business',
+  displayName: GetCorpFullDescription(CorpTypeCd.PARTNERSHIP),
+  nameRequestType: NameRequestEntityTypes.GP,
+  addressLabel: 'Business Addresses',
```

Comment:
I think addressLabel should be 'Record Conversion' according to the new [wireframes design] (https://preview.uxpin.com/363a49b33bdb2a314ccac0f18daad8969e93b337#/pages/149273920/simulate/sitemap).